packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -49,13 +49,18 @@ class FlutterExtension {
     /** Sets the minSdkVersion used by default in Flutter app projects. */
     static int minSdkVersion = 19
 
-    /** Sets the targetSdkVersion used by default in Flutter app projects. */
+    /**
+     * Sets the targetSdkVersion used by default in Flutter app projects.
+     * Should always be the latest available version.
```

Comment:
Latest available *stable* version. 
Also nit, this should probably be a sentence. 
```suggestion
     * targetSdkVersion should be the latest available stable version. 
```

Reply:
changed to suggested